specification/appconfiguration/AppConfiguration/routes.tsp

```diff
@@ -87,106 +52,50 @@ op getKeys is Foundations.ResourceList<
 @doc("Requests the headers and status of the given resource.")
 @route("/keys")
 @head
-op checkKeys is appConfigOperation<
-  {
-    @doc("A filter for the name of the returned keys.")
-    @query("name")
-    name?: string;
-
-    ...syncTokenHeader;
-
-    @doc("""
-      Instructs the server to return elements that appear after the element referred
-      to by the specified token.
-      """)
-    @query("After")
-    after?: string;
-
-    @doc("""
-      Requests the server to respond with the state of the resource at the specified
-      time.
-      """)
-    @header("Accept-Datetime")
-    acceptDatetime?: string;
-  },
-  OkResponse & {
-    ...syncTokenHeader;
-  }
+op checkKeys is AppConfigOperation<
+  KeysRequestParams,
+  OkResponse & SyncTokenHeader
 >;
 
+alias KeyValuesRequestParams = {
+  ...KeyQueryParam<"""
```

Comment:
I know it is painful right now to apply customized doc string for spread properties, though do they all need doc customization to build into the spread models?  If majority doesn't need customization with few exceptions, you can always use `@@doc` to do targeted overwrite of doc strings.

Reply:
I attempted to follow this [example](https://typespec.io/playground/?c=bW9kZWwgQmFzZSB7DQogIC8qKiBhxQIqL8QQYTogc3RyaW5nOw0KxBBAZG9jKCJiIGIgYiIpxBFiyyF9xCQvKioNCiAqIEBwcm9wIGEgMcUByxNiIDLFAcQTLw0K5gCCRm9v5gCBLi4u5ACOxk4%3D&e=%40typespec%2Fopenapi3&options=%7B%7D), but adding the doc over an alias doesnt seem to be allowed currently. I tried compiling and there was no change to the generated openapi spec. Also, we cant to `@@doc` on all of the params where we're doing this since the docs are specific to certain operations through anonymous models. Let me know if you have any ideas for how to fix this. I'll merge this PR in now though so the service team can leverage this refactored code.